homeassistant/components/vicare/climate.py

```diff
@@ -146,16 +146,16 @@ class ViCareClimate(ClimateEntity):
     _attr_max_temp = VICARE_TEMP_HEATING_MAX
     _attr_target_temperature_step = PRECISION_WHOLE
     _attr_preset_modes = list(HA_TO_VICARE_PRESET_HEATING)
+    _attributes: dict[str, Any] = {}
```

Comment:
Oh right, you can move stuff out of the constructor except when they are a dict. It will cause strange bugs, so we do create this in the constructor